bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -49,6 +50,23 @@ spec:
         - name: "mariadb-galera"
           image: {{ template "mariadb-galera.image" . }}
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          command:
+            - bash
+            - -ec
+            - |
+                # Bootstrap from the indicated node
+                HOSTNAME="$(hostname -s)"
+                if [[ "$HOSTNAME" =~ (.*)-([0-9]+)$ ]]; then
+                    NODE="${BASH_REMATCH[2]}"
+                    if [[ "$NODE" -eq "{{ int .Values.galera.bootstrap.bootstrapFromNode }}" ]]; then
+                        export MARIADB_GALERA_CLUSTER_BOOTSTRAP=yes
+                        export MARIADB_GALERA_FORCE_SAFETOBOOTSTRAP={{ ternary "yes" "no" .Values.galera.bootstrap.forceSafeToBootstrap }}
+                    fi
+                else
+                    echo "Failed to get index from hostname $HOST"
+                    exit 1
+                fi
```

Comment:
This else is not necessary with my suggestion

Reply:
removed